bin/analyze_hoc_activity

```diff
@@ -88,7 +88,20 @@ def main()
     hoc_company_totals: hoc_company_totals,
   }
 
-  lines_of_code = rack_env?(:production) ? `#{deploy_dir('bin','count-lines-of-code-from-milestone-logs_v2')}`.to_i : 0
+  # In the production environment, begin with a known previous value in case we
+  # are unable to obtain a more recent number. The 11151730618 number is from
+  # 2016-01-04.
+  lines_of_code = rack_env?(:production) ? 11151730618 : 0
+  # Since s3 (which the binary relies on) may fail, we retry a few times hoping
+  # for success.
+  3.times do
+    lines_of_code_from_milestone = `#{deploy_dir('bin','count-lines-of-code-from-milestone-logs_v2')}`.to_i
```

Comment:
this is a behavior change -- in the past we didn't try to run 'count-lines-of-code-from-milestone-logs_v2' in non-prod envs. There's probably a reason for this (doesn't work in non-prod environments? not running an expensive script because we don't actually care?) so I think we should keep the old behavior unless there's a good reason not to.